libs/hmpb/src/marking.ts

```diff
@@ -0,0 +1,199 @@
+import PdfDocument from 'pdfkit';
+
+import { assert } from '@votingworks/basics';
+import {
+  ballotPaperDimensions,
+  Election,
+  GridPosition,
+  Rect,
+  Vote,
+  VotesDict,
+} from '@votingworks/types';
+
+import {
+  BUBBLE_HEIGHT_PX,
+  BUBBLE_WIDTH_PX,
+  pageMarginsInches,
+  TIMING_MARK_DIMENSIONS,
+} from './ballot_components';
+
+// NOTE: All values used in this module are in PDF user space `pt` units.
+
+const PT = 1;
+const IN = 72 * PT;
+const PX = IN / 96;
+
+const pageMargins = [
+  pageMarginsInches.left * IN,
+  pageMarginsInches.top * IN,
+] as const;
+
+const timingMarkCount = [34, 41] as const;
+const timingMarkSize = [
+  TIMING_MARK_DIMENSIONS.width * IN,
+  TIMING_MARK_DIMENSIONS.height * IN,
+] as const;
+
+const bubbleSize = [BUBBLE_WIDTH_PX * PX, BUBBLE_HEIGHT_PX * PX] as const;
+// We grow the mark slightly to help make up for slight variations in printing:
```

Comment:
Does this overfill the bubble? Is that preferable, either for aesthetics or interpretation, to slightly underfilling the bubble?

Reply:
Given our mark thresholds seems like an underfill would be fine, but if overfilling doesn't mess with interpretation I agree it's better aesthetically / inspires more confidence